From 16d489174ef15e2ec42570a24d49211fa9bd834d Mon Sep 17 00:00:00 2001
From: Ryan QIAN <b32804@freescale.com>
Date: Wed, 7 Mar 2012 10:56:13 +0800
Subject: ENGR00176279 [MX6DL] MMC: improve SD3.0 SDR104 mode compatability.

remove SION bit for SD3_CMD pad control, it will enlarge clock tuning
window on MX6DL.

Signed-off-by: Ryan QIAN <b32804@freescale.com>
---
 arch/arm/plat-mxc/include/mach/iomux-mx6dl.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/plat-mxc/include/mach/iomux-mx6dl.h b/arch/arm/plat-mxc/include/mach/iomux-mx6dl.h
index 6d0a53b..918dc58 100644
--- a/arch/arm/plat-mxc/include/mach/iomux-mx6dl.h
+++ b/arch/arm/plat-mxc/include/mach/iomux-mx6dl.h
@@ -3385,7 +3385,7 @@
 #define MX6DL_PAD_SD3_CMD__USDHC3_CMD_100MHZ                                   \
 		IOMUX_PAD(0x06F8, 0x0310, 0 | IOMUX_CONFIG_SION, 0x0000, 0, MX6DL_USDHC_PAD_CTRL_100MHZ)
 #define MX6DL_PAD_SD3_CMD__USDHC3_CMD_200MHZ                                   \
-		IOMUX_PAD(0x06F8, 0x0310, 0 | IOMUX_CONFIG_SION, 0x0000, 0, MX6DL_USDHC_PAD_CTRL_200MHZ)
+		IOMUX_PAD(0x06F8, 0x0310, 0, 0x0000, 0, MX6DL_USDHC_PAD_CTRL_200MHZ)
 #define MX6DL_PAD_SD3_CMD__UART2_CTS                                           \
 		IOMUX_PAD(0x06F8, 0x0310, 1, 0x0000, 0, MX6DL_UART_PAD_CTRL)
 #define MXSDL_PAD_SD3_CMD__UART2_RTS                                          \
-- 
1.7.9.5

